src/quick-lint-js/fe/lex.cpp

```diff
@@ -2033,7 +2033,13 @@ void lexer::skip_block_comment() {
 }
 
 void lexer::skip_line_comment_body() {
-#if QLJS_HAVE_X86_SSE2
+#if QLJS_HAVE_ARM_NEON
+  using bool_vector = bool_vector_16_neon;
+  using char_vector = char_vector_16_neon;
+#elif QLJS_HAVE_WEB_ASSEMBLY_SIMD128
+  using bool_vector = bool_vector_16_wasm_simd128;
+  using char_vector = char_vector_16_wasm_simd128;
+#elif QLJS_HAVE_X86_SSE2
```

Comment:
I also added the options for ARM Neon and Web Assembly SIMD128

Reply:
Oh, I didn't even notice that AArch64 wasn't SIMD-optimized. I guess why the performance win wasn't that drastic (9%). (But I'm surprised there was a win at all, given the slowdown for no-SIMD on x86_64.)